mapswipe_workers/mapswipe_workers/utils/slack_helper.py

```diff
@@ -57,24 +57,26 @@ def send_progress_notification(project_id: int):
     """Send progress notification to project managers in Slack."""
     fb_db = auth.firebaseDB()
     progress = fb_db.reference(f"v2/projects/{project_id}/progress").get()
-    project_name = fb_db.reference(f"v2/projects/{project_id}/name").get()
-    notification_90_sent = fb_db.reference(
-        f"v2/projects/{project_id}/notification_90_sent"
-    ).get()
-    notification_100_sent = fb_db.reference(
-        f"v2/projects/{project_id}/notification_100_sent"
-    ).get()
-    logger.info(
-        f"{project_id} - progress: {progress},"
-        f"notifications: {notification_90_sent} {notification_100_sent}"
-    )
 
-    if progress >= 90 and not notification_90_sent:
-        # send notification and set value in firebase
-        fb_db.reference(f"v2/projects/{project_id}/notification_90_sent").set(True)
-        send_slack_message("notification_90", project_name, project_id)
+    if progress >= 90:
```

Comment:
I missed this the first time, but what data type is `fb_db.....get()` returning? I imagine a string? python might be unhappy about the comparison.

Reply:
.get() usually returns a list or ordered dict.

Reply:
in this case it returns an integer in this case, if if is a number in firebase.